{"name": "test_getting_user_list", "status": "broken", "statusDetails": {"message": "AttributeError: 'Responce' object has no attribute 'response_status'. Did you mean: 'responce_status'?", "trace": "say_hello = None, make_random_number = 84\ncreate_fake_user = {'address': '32434 Brian Views Apt. 290\\nNew Stephanie, MP 87675', 'email': 'patricia56@example.com', 'name': 'Lauren', 'phone_number': '001-589-846-5784x510', ...}\n\n    @pytest.mark.regression\n    @allure.feature('User API')\n    @allure.story('Get user list')\n    @allure.step(\"Получение списка пользователей\")\n    def test_getting_user_list(say_hello: None, make_random_number: int, create_fake_user: dict[str, str]):\n        with allure.step(\"Шаг 1: Запрос списка пользователей\"):\n            headers = { \"Accept\": \"application/json\"  # Указываем, что ожидаем ответ в формате JSON}\n            }\n            resp = requests.get(SERVICE_URL, headers = None)\n            test_object = Responce(resp)\n        #print(make_random_number, create_fake_user, \"-from test\")\n        with allure.step(\"Шаг 2: Проверка статуса ответа\"):\n            #print(\"->\", test_object.get_parsed_json())\n            test_object.validate(ApiResponse)\n>           test_object.assert_status_code(200)#.validate(User)\n\ntests/something2_test.py:31: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <src.baseclasses.responce.Responce object at 0x109a879a0>, status_code = 200\n\n    def assert_status_code(self, status_code):\n        if isinstance(status_code, list):\n            assert self.responce_status in status_code, f\"{GlobalErrorMessages.WRONG_STATUS_CODE.value}, got {self.response_status}\"\n        else:\n>           assert self.responce_status == status_code, f\"{GlobalErrorMessages.WRONG_STATUS_CODE.value}, got {self.response_status}\"\nE           AttributeError: 'Responce' object has no attribute 'response_status'. Did you mean: 'responce_status'?\n\nsrc/baseclasses/responce.py:38: AttributeError"}, "steps": [{"name": "Получение списка пользователей", "status": "broken", "statusDetails": {"message": "AttributeError: 'Responce' object has no attribute 'response_status'. Did you mean: 'responce_status'?\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/Users/medstar/chkpytest/tests/something2_test.py\", line 31, in test_getting_user_list\n    test_object.assert_status_code(200)#.validate(User)\n  File \"/Users/medstar/chkpytest/src/baseclasses/responce.py\", line 38, in assert_status_code\n    assert self.responce_status == status_code, f\"{GlobalErrorMessages.WRONG_STATUS_CODE.value}, got {self.response_status}\"\n"}, "steps": [{"name": "Шаг 1: Запрос списка пользователей", "status": "passed", "start": 1725261098963, "stop": 1725261138157}, {"name": "Шаг 2: Проверка статуса ответа", "status": "broken", "statusDetails": {"message": "AttributeError: 'Responce' object has no attribute 'response_status'. Did you mean: 'responce_status'?\n", "trace": "  File \"/Users/medstar/chkpytest/tests/something2_test.py\", line 31, in test_getting_user_list\n    test_object.assert_status_code(200)#.validate(User)\n  File \"/Users/medstar/chkpytest/src/baseclasses/responce.py\", line 38, in assert_status_code\n    assert self.responce_status == status_code, f\"{GlobalErrorMessages.WRONG_STATUS_CODE.value}, got {self.response_status}\"\n"}, "start": 1725261138157, "stop": 1725261138157}], "parameters": [{"name": "say_hello", "value": "None"}, {"name": "make_random_number", "value": "84"}, {"name": "create_fake_user", "value": "{'name': 'Lauren', 'surname': 'Lewis', 'email': 'patricia56@example.com', 'phone_number': '001-589-846-5784x510', 'address': '32434 Brian Views Apt. 290\\nNew Stephanie, MP 87675'}"}], "start": 1725261098962, "stop": 1725261138160}], "start": 1725261098962, "stop": 1725261138165, "uuid": "71a48191-797b-4c42-986c-efafa6172e0e", "historyId": "8916657901d854c3a6aca0a0a4bbf70a", "testCaseId": "8916657901d854c3a6aca0a0a4bbf70a", "fullName": "tests.something2_test#test_getting_user_list", "labels": [{"name": "story", "value": "Get user list"}, {"name": "feature", "value": "User API"}, {"name": "tag", "value": "regression"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "something2_test"}, {"name": "host", "value": "MS105M.local"}, {"name": "thread", "value": "7214-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.something2_test"}]}